packages/create-react-app-demo/src/basic-chat/index.tsx

```diff
@@ -1,46 +1,58 @@
-import React from 'react';
-import { AIRoot, ChatMessage, conversationAtom, modelCallInProgress } from './ai.tsx';
-import { useAtom } from 'jotai';
+/** @jsxImportSource ai-jsx/react */
+import * as AI from 'ai-jsx/react';
+import { useAI } from 'ai-jsx/react';
```

Comment:
You can get rid of this import and do `AI.useAI` below.

Reply:
not sure I even am using useAI, I just removed it